Makefile

```diff
@@ -69,21 +70,18 @@ tag_release: tagz.py
 	$(eval RELEASE_DATE := $(shell $(PYTHON) -c 'import datetime; now = datetime.datetime.utcnow(); tue = now + datetime.timedelta(days=(1 - now.weekday()) % 7); print tue.strftime("%Y.%m.%d")'))
 	$(PYTHON) tagz.py -r mozilla/solitude,mozilla/spartacus,mozilla/webpay,mozilla/commbadge,mozilla/fireplace,mozilla/marketplace-operator-dashboard,mozilla/marketplace-stats,mozilla/monolith-aggregator,mozilla/transonic,mozilla/zamboni -c create -t $(RELEASE_DATE)
 
-deploy_release:
+deploy_release: check_deploy_env
 	$(eval RELEASE_DATE := $(shell $(PYTHON) -c 'import datetime; now = datetime.datetime.utcnow(); tue = now + datetime.timedelta(days=(1 - now.weekday()) % 7); print tue.strftime("%Y.%m.%d")'))
-	$(PYTHON) scripts/dreadnot-deploy.py			\
-	-c dreadnot-stage.ini -e stage -r $(RELEASE_DATE)	\
-		payments.allizom.org-solitude			\
-		payments-proxy.allizom.org-solitude		\
-                marketplace.allizom.org-spartacus               \
-		marketplace.allizom.org-webpay			\
-		monolith.allizom.org-aggregator			\
-		marketplace.allizom.org-marketplace-operator-dashboard	\
-		marketplace.allizom.org-marketplace-stats	\
-		marketplace.allizom.org-commbadge		\
-                marketplace.allizom.org-transonic               \
-		marketplace.allizom.org-fireplace		\
-		marketplace.allizom.org-zamboni
-
+	curl -k -X POST $(JENKINS_URL) \
```

Comment:
Is the `-k` b/c of a home built cert? It'll be cool when Mozilla does that free SSL cert stuff they announced.